gen/issue185_test.go

```diff
@@ -0,0 +1,308 @@
+package gen
```

Comment:
Isn't this file just a duplicate of the one in the root of the file tree? Remove it?

Reply:
Oops! Done.